runtime/buildtools.mk

```diff
@@ -183,6 +183,12 @@ OMRGLUE_INCLUDES = \
   ../gc_trace \
   ../gc_vlhgc
 
+#generate jvmti.h
+include/jvmti.h: include/jvmti.h.m4
+	m4 -D JAVA_SPEC_VERSION=$(VERSION_MAJOR) $< > $@
```

Comment:
This rule should move to `runtime/include/module.xml` so that we don't have duplication for cmake.